elements/designer-app/designer-app.html

```diff
@@ -28,14 +29,24 @@
 <dom-module id="designer-app">
 
   <style>
+    :root {
+      --dark-primary-color: var(--paper-grey-900);
+      --default-primary-color: var(--paper-grey-800);
+      --light-primary-color: var(--paper-grey-500);
+      --accent-color: var(--paper-yellow-a200);
+      --primary-background-color: var(--paper-grey-200);
+      --primary-text-color: rgba(255, 255, 255, var(--dark-primary-opacity));
```

Comment:
These `rgba`s look like they would be useful to have in `polymerelements/paper-styles`.